src/wp-includes/class-wp-scripts.php

```diff
@@ -350,7 +350,16 @@ public function do_item( $handle, $group = false ) {
 
 		$translations = $this->print_translations( $handle, false );
 		if ( $translations ) {
-			$translations = wp_get_inline_script_tag( $translations, array( 'id' => "{$handle}-js-translations" ) );
+			/**
+			 * The sourceURL comment is not included by @see WP_Scripts::print_translations()
+			 * when `$display == false` to prevent issues where the script tag contents are used
+			 * by extenders for other purposes, for example concatenated with other script content.
```

Comment:
Since this isn't the context for a docblock, I don't believe `/**` is the right comment style. Also, the bare `@see` isn't right. If used, I believe it would need to be `{@see WP_Scripts::print_translations()}` but @peterwilsoncc has informed me this shouldn't be used for functions.

```suggestion
			/*
			 * The sourceURL comment is not included by WP_Scripts::print_translations()
			 * when `$display === false` to prevent issues where the script tag contents are used
			 * by extenders for other purposes, for example concatenated with other script content.
```

Reply:
You're right, the inline form of the see tag should be in braces `{@see …}`.

In my editor, I get proper highlighting and go-to definition if this is a PHPDoc comment (starts with `/**`) and I use the `@see` tag.

I've updated the comment. I can adjust further if necessary, but the best experience for me seems to be using `/** {@see …} */`.

Reply:
@sirreal Yeah, it should only be a single asterisks as the parser can confuse the inline comment with a docblock, see https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-2-multi-line-comments

Reply:
Done, thanks for the clarification.